src/main/java/unibook/model/util/SampleDataUtil.java

```diff
@@ -22,25 +22,29 @@
     public static Person[] getSamplePersons() {
         return new Person[] {
             new Person(new Name("Alex Yeoh"), new Phone("87438807"), new Email("alexyeoh@example.com"),
-                getTagSet("friends"), getModuleSet("CS2103")),
+                getTagSet("friends"), getModuleSet("CS2106")),
             new Person(new Name("Bernice Yu"), new Phone("99272758"), new Email("berniceyu@example.com"),
                 getTagSet("colleagues", "friends"), getModuleSet("CS2103")),
             new Person(new Name("Charlotte Oliveiro"), new Phone("93210283"), new Email("charlotte@example.com"),
                 getTagSet("neighbours"), getModuleSet("CS2103")),
-            new Person(new Name("David Li"), new Phone("91031282"), new Email("lidavid@example.com"),
-                getTagSet("family"), getModuleSet("CS2103")),
-            new Person(new Name("Irfan Ibrahim"), new Phone("92492021"), new Email("irfan@example.com"),
-                getTagSet("classmates"), getModuleSet("CS2103")),
-            new Person(new Name("Roy Balakrishnan"), new Phone("92624417"), new Email("royb@example.com"),
-                getTagSet("colleagues"), getModuleSet("CS2103"))
         };
     }
 
+    public static Module[] getSampleModules() {
+        Module testModule1 = new Module(new ModuleName("Software Engineering"), new ModuleCode("CS2103"));
+        Module testModule2 = new Module(new ModuleName("Introduction to Operating Systems"), new ModuleCode("CS2106"));
+
+        return new Module[] {testModule1, testModule2};
+    }
+
     public static ReadOnlyUniBook getSampleUniBook() {
         UniBook sampleAb = new UniBook();
         for (Person samplePerson : getSamplePersons()) {
             sampleAb.addPerson(samplePerson);
         }
+        for (Module mod : getSampleModules()) {
+            sampleAb.addModule(mod);
+        }
         return sampleAb;
     }
```

Comment:
call get samplemodules first, then get sample people, here you need to ensure bidirectionality
for each module, add each person in getSamplePersons who is in that module, for now just add all to student list.
for each person, after all the persons are created, use the modules created before to add them into the correct person's module list.